xplot.py

```diff
@@ -1 +1,49 @@
-#
+'''
+Problem 1: Plot Bar
+'''
+# TEST CASE GENERATOR # DO NOT MODIFY # 
+inputs_list = [["A", 627.2],["B", 38.9],["C", 11.3]]
+inputs_dict = {"A": 627.2, "B": 38.9,"C": 11.3}
+
+# SOLUTION
+
+def barGraph1(inputs_list):
+  max_val = max(inputs_list, key=lambda x: x[1])[1]
+  for element in inputs_list:
+      print(element[0],int((element[1]/max_val)*50)*"#")
```

Comment:
can do something like:

```
for i, j in inputs_list:
    print(i, (j * 50 // max_val) * '#')
```

Reply:
This won't work - you're multiplying a string by a float, so will return a type error. 
`(j * 50 // max_val)` is a float, need an int() around that expression, like this:

for i, j in inputs_list:
    print(i, int(j * 50 // max_val) * '#')

Thanks for the suggestion of using i,j over element[0], element[1], looks cleaner this way.

Reply:
I believe // yields an integer, not a float, so this should work

Reply:
If either a or b is a float in a//b, will return a float.
In this case, the numerator is a float. When tested, this returns type error.